[2024-01-24T01:13:13.719+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tutorial_dag.not_valid scheduled__2024-01-23T23:30:00+00:00 [queued]>
[2024-01-24T01:13:13.727+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tutorial_dag.not_valid scheduled__2024-01-23T23:30:00+00:00 [queued]>
[2024-01-24T01:13:13.727+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-01-24T01:13:13.738+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): not_valid> on 2024-01-23 23:30:00+00:00
[2024-01-24T01:13:13.742+0000] {standard_task_runner.py:60} INFO - Started process 166 to run task
[2024-01-24T01:13:13.744+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'tutorial_dag', 'not_valid', 'scheduled__2024-01-23T23:30:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/tutorial_dag.py', '--cfg-path', '/tmp/tmptabo4upe']
[2024-01-24T01:13:13.745+0000] {standard_task_runner.py:88} INFO - Job 18: Subtask not_valid
[2024-01-24T01:13:13.781+0000] {task_command.py:423} INFO - Running <TaskInstance: tutorial_dag.not_valid scheduled__2024-01-23T23:30:00+00:00 [running]> on host 4e1f8651ba76
[2024-01-24T01:13:13.839+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tutorial_dag' AIRFLOW_CTX_TASK_ID='not_valid' AIRFLOW_CTX_EXECUTION_DATE='2024-01-23T23:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-23T23:30:00+00:00'
[2024-01-24T01:13:13.840+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-01-24T01:13:13.841+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Quantity NOT OK'"]
[2024-01-24T01:13:13.847+0000] {subprocess.py:86} INFO - Output:
[2024-01-24T01:13:13.848+0000] {subprocess.py:93} INFO - Quantity NOT OK
[2024-01-24T01:13:13.849+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-24T01:13:13.867+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=tutorial_dag, task_id=not_valid, execution_date=20240123T233000, start_date=20240124T011313, end_date=20240124T011313
[2024-01-24T01:13:13.916+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-24T01:13:13.927+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
